packages/mdx/index.js

```diff
@@ -1,12 +1,12 @@
 /**
- * @typedef {import('./lib/util/resolve-evaluate-options.js').Fragment} Fragment
- * @typedef {import('./lib/util/resolve-evaluate-options.js').Jsx} Jsx
- * @typedef {import('./lib/util/resolve-evaluate-options.js').JsxDev} JsxDev
+ * @typedef {import('hast-util-to-jsx-runtime').Fragment} Fragment
+ * @typedef {import('hast-util-to-jsx-runtime').Jsx} Jsx
+ * @typedef {import('hast-util-to-jsx-runtime').JsxDev} JsxDev
  * @typedef {import('./lib/util/resolve-evaluate-options.js').UseMdxComponents} UseMdxComponents
  * @typedef {import('./lib/compile.js').CompileOptions} CompileOptions
  * @typedef {import('./lib/core.js').ProcessorOptions} ProcessorOptions
- * @typedef {import('./lib/evaluate.js').EvaluateOptions} EvaluateOptions
- * @typedef {import('./lib/run.js').RunOptions} RunOptions
+ * @typedef {import('./lib/util/resolve-evaluate-options.js').EvaluateOptions} EvaluateOptions
+ * @typedef {import('./lib/util/resolve-evaluate-options.js').RunOptions} RunOptions
  */
```

Comment:
I like an `index.d.ts`, as it has `export type {} from`, here it would get rid of the typedef/type alias  https://github.com/rehypejs/rehype-document/blob/main/index.d.ts

Reply:
I’m working on a follow-up PR where I think this is better discussed. I do agree `export {} from` is an improvement over a typedef.